src/main/java/org/wso2/carbon/connector/twilio/conference/GetConferenceList.java

```diff
@@ -42,8 +42,32 @@ public void connect(MessageContext messageContext) throws ConnectException {
         SynapseLog log = getLog(messageContext);
         log.auditLog("Start: get conference list");
 
-        // optional parameters for filtering resultant conferences.
-        // See http://www.twilio.com/docs/api/rest/conference#list-get-filters
+        try {
+            TwilioRestClient twilioRestClient = TwilioUtil.getTwilioRestClient(messageContext);
+            Request request = new Request(HttpMethod.GET, Domains.API.toString(),
+                    "/" +
+                    TwilioUtil.API_VERSION +
+                    "/" +
+                    TwilioUtil.API_ACCOUNTS +
+                    "/" +
+                    twilioRestClient.getAccountSid() +
+                    "/" +
+                    TwilioUtil.API_CONFERENCES);
+            addQueryParams(request, messageContext);
+            Response response = twilioRestClient.request(request);
+
+            OMElement omResponse = TwilioUtil.parseResponse(response);
+            TwilioUtil.preparePayload(messageContext, omResponse);
+        } catch (Exception e) {
+            log.error(e.getMessage());
```

Comment:
Do we need to both log and throw the error?

Reply:
Do the new methods throw any exceptions? Previously the `TwilioUtil.getTwilioRestClient` was throwing an exception and in the current implementation I couldn't find any.

Reply:
I think throws and try catch were expected to handle twilioRestClient.request and TwilioUtil.getTwilioRestClient methods but given that they are no longer there we can remove those